use-cases/README.md

```diff
@@ -0,0 +1,90 @@
+# Dapr use cases
+
+This document briefly describes scenarios that Dapr enables for your microservices. Each use case is accompanied with a link to learn more details.
+
+## Service invocation
+
+Dapr provides a uniform API that your microservice can use to find and reliably communicate with other microservices in your system. By using this API, your code can seamlessly take advantage of several built-in features:
+
+- HTTP and [gRPC](https://grpc.io) support
+- Robust retry logic
+- Intelligent error handling
+- Distributed tracing
+
+Visit the following documents to learn more:
+
+- [More details on service invocation](https://github.com/dapr/docs/blob/master/concepts/service-invocation/README.md)
+- [How to get started with service invocation](https://github.com/dapr/docs/tree/master/howto/invoke-and-discover-services)
+
+
+## State management
+
+Dapr provides an abstraction API that abstracts over [many different databases](https://github.com/dapr/docs/blob/master/howto/setup-state-store/supported-state-stores.md). Your microservice can leverage these APIs to utilize any supported database without finding or adding a third party SDK to your codebase.
+
+Visit the following documents to learn more:
+
+- [More details on state management](https://github.com/dapr/docs/blob/master/concepts/state-management/README.md)
+- [How to get started with state management](https://github.com/dapr/docs/tree/master/howto/setup-state-store)
+
+## Pub/Sub
```

Comment:
Even if the Dapr building block is named PubSub - I always ask myself, if "asynchronous messaging" would not be a better term. @msfussell what are your thoughts about that in general?

Reply:
I agree. Alternatively, if you still want to keep `pub/sub`, I can change this title to: `Publish / Subscribe asynchronous messaging`, so you have both in here. Let me know what you prefer

Reply:
Async messaging includes more than pub/sub, for example - message queues. In the current architecture pub/sub does not cover that, you would use queues with bindings.

Reply:
I'm going to break this use cases document out in appropriate READMEs under the `concepts/` directory. I'll follow the terminology inside of the pub sub concept for now. Happy to update it in a follow-up PR if that's ok.